README.md

```diff
@@ -49,6 +49,62 @@ const server = new InversifyExpressServer(container, null);
 })
 ```
 
+## What is SiberMetrics and how it works.
+
+- Record metrics from HTTP Request
+- Metrics exporter
+
+Create an instance of the SiberMetrics in your server:
+
+```ts
+export const Metrics = new SiberMetrics(env);
+```
+
+SiberMetrics allow you to record `success` or `failure` requests. You can either record metrics in your `Controller` or `Error` handler using its `record` method:
+
+```ts
+Metrics.record(req, res);
+```
+
+Also siberMetrics allow the export of all recorded metrics to prometeus server through the `send` method:
+
+Create an HTTP Handler to expose your server metrics to prometeus server.
+
+```ts
+app.get("/metrics", Metrics.send.bind(Metrics));
+```
+
+## Parsing your environment variables
+
+Siber `autoloadEnv` function helps to parse and validate server environment variables.
+
+# Usage
+To use `autoloadEnv` you need to define;
+
+- The `type` definition of your server environment configurations
+- joi validation schema for the `type` definition
+
+```ts
+import joi from "@hapi/joi";
+import { autoloadEnv, siberConfig, mongoConfig, redisConfig } from "@random-guys/siber";
+
+interface EnvironmentConfig {
+  any_variable: string
+}
+
+export const env = autoloadEnv<EnvironmentConfig>(
+  siberConfig({
+    ...mongoConfig,
+    ...redisConfig,
+    any_variable: joi.string().required(),
+  })
+);
```

Comment:
this is wrong. Your type only supports one env variable, but the schema you're defining contains at least 10.